Ktisis/Util/GuiHelpers.cs

```diff
@@ -11,26 +11,30 @@
 
 using FFXIVClientStructs.Havok;
 
-using Ktisis.Structs.Bones;
-
 namespace Ktisis.Util
 {
 	internal class GuiHelpers {
-		public static bool IconButtonHoldConfirm(FontAwesomeIcon icon, string tooltip, bool isHoldingKey) {
+		public static bool IconButtonHoldConfirm(FontAwesomeIcon icon, string tooltip, bool isHoldingKey, Vector2 size = default) {
```

Comment:
This is going to result in a size of `(0, 0)`... is that intentional?

Reply:
Yes, a size of `(0, 0)` means it's going to take the space it needs.

I wanted to keep the code clean without too much ifs and it seems this works as well as `(-1, -1)`, which was my first intention.